app/texters/booking_texter.rb

```diff
@@ -0,0 +1,7 @@
+class BookingTexter
+  def confirmed(*recipients)
```

Comment:
This class is incomplete and untested. I was trying to have a similar interface between mailers and texters to allow us to interchange instances in the controller based on the `confirmation_type`. A sort of strategy pattern.